src/components/ProductFullInformation.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+import { products } from "@components/Products";
+import axios from "axios";
+import { useParams } from "react-router-dom";
+import "@styles/ProductFullInformation.scss";
+
+const ProductFullInformation = (): JSX.Element => {
```

Comment:
не совсем корректная типизация
```suggestion
const ProductFullInformation: React.FC = () => {
```